package.json

```diff
@@ -36,6 +36,7 @@
     "clean": "rimraf .next",
     "lint": "eslint pages fusion",
     "pre-commit": "npm run lint",
+    "test":"npm run lint",
```

Comment:
+1 

This will eventually also have tests being run. Adding the

Reply:
yes, that has to be done at the Travis CI level.

Reply:
Also technically any PR from feature has to go into dev and then move it to master from dev.